### ISSUE TITLE
2.5.2 EmuHawk crashes on startup (XINPUT_STATE)

### ISSUE THREAD

**Issue by Meerkov** [TASVideos/BizHawk#2537]
System.Runtime.InteropServices.SEHException (0x80004005): External component has thrown an exception.
   at XInputGetState(UInt32 , _XINPUT_STATE* )
   at SlimDX.XInput.Controller.get_IsConnected()
   at BizHawk.Client.EmuHawk.GamePad360.Initialize()
   at BizHawk.Client.EmuHawk.DirectInputAdapter.FirstInitAll(IntPtr mainFormHandle)
   at BizHawk.Client.EmuHawk.MainForm..ctor(String[] args)
   at BizHawk.Client.EmuHawk.Program.SubMain(String[] args)

Basically this is a fresh windows machine, with a fresh download of 2.5.2. I had a game pad plugged in, but unplugging it and trying again resulted in the same error.

**Comment by TiKevin83** [comment]
try installing latest prereqs
https://github.com/TASVideos/BizHawk-Prereqs/releases

**Comment by Meerkov** [comment]
This worked. Any chance this can work as a Feature Request to give a more helpful hint in the exception?

**Comment by YoshiRulz** [comment]
I'd rather make OpenTK input the default than figure out how to catch DirectX exceptions
